app/src/main/res/layout/fragment_login_detail.xml

```diff
@@ -33,10 +33,12 @@
     <TextView
```

Comment:
Even though it's rare, there could also be passwords longer than 1 line. Maybe we can change that too to allow multi-line.

Reply:
After the discussion we had, it makes sense to not change it also for passwords, since the UI will be a bit weird.